pkg/kapp/logs/pod_container_log.go

```diff
@@ -44,6 +44,30 @@ func NewPodContainerLog(
 }
 
 func (l PodContainerLog) Tail(ui ui.UI, cancelCh chan struct{}) error {
+	for {
+		linePrefix := ""
+		if len(l.opts.LinePrefix) > 0 {
+			linePrefix = l.opts.LinePrefix + " | "
+		}
+		err := l.StartTail(ui, cancelCh)
+
+		if err == io.EOF {
+			if l.opts.Follow {
+				ui.BeginLinef("Container Restarted\n")
```

Comment:
we follow same format here e.g. `ui.BeginLinef("%s# container restarting '%s' logs\n", linePrefix, l.tag)`

Reply:
Adopted